agent/pkg/oas/oas_generator.go

```diff
@@ -160,24 +159,28 @@ func (g *defaultOasGenerator) getGen(dest string, urlStr string) *SpecGen {
 	val, found := g.serviceSpecs.Load(dest)
 	var gen *SpecGen
 	if !found {
-		gen = NewGen(u.Scheme + "://" + dest)
+		serviceAddress := u.Scheme + "://" + dest
+		logger.Log.Infof("Create SpecGen for service %s", serviceAddress)
```

Comment:
I don't think we need this log, it could be spamming

Reply:
https://github.com/up9inc/mizu-enterprise/pull/137/commits/29a819e512052c3d982da7fb82f5fdd5b48b51a3